### PR TITLE
testing/akregator: new aport

### PR DESCRIPTION
**Pull request by PureTryOut**

### CODE REVIEW

File: testing/akregator/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Contributor: Bart Ribbers <bribbers@disroot.org>
+# Maintainer: Bart Ribbers <bribbers@disroot.org>
+pkgname=akregator
+pkgver=19.04.2
+pkgrel=0
+arch="all !ppc64le !s390x" # Limited by qt5-qtwebengine
+url="https://kontact.kde.org/components/akregator.html"
+pkgdesc="RSS Feed Reader "
```
Comment (maxice8) [inline testing/akregator/APKBUILD]: There is an extra space at the end

**Comment by maxice8** [comment]
Please notify when proposed changes are made so i am contacted by e-mail and can react quickly

**Comment by PureTryOut** [comment]
Oh sorry, I thought pressing the "Resolve conversation" button would do that. Will do in the future.

**Comment by maxice8** [comment]
On Mon, Jun 24, 2019 at 05:21:46AM -0700, Bart Ribbers wrote:
> Oh sorry, I thought pressing the "Resolve conversation" button would do that. Will do in the future.
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/alpinelinux/aports/pull/9038#issuecomment-504986306

No worries, thanks for understasnding

**Pull request closed without merging**